bims/api_views/site_in_country.py

```diff
@@ -14,27 +16,26 @@ class SiteInCountry(LoginRequiredMixin, APIView):
     def get(self, request):
         lat = request.GET.get('lat', None)
         lon = request.GET.get('lon', None)
+
         if not lat or not lon:
             raise Http404('Missing coordinates')
-        base_country_code = preferences.SiteSetting.base_country_code
-        if not base_country_code:
-            return Response(True)
+        url = (
```

Comment:
@Fanevanjanahary can you add this url in site setting, so when we deploy to different instance we don't need to update the code.